Source/SwiftLintCore/Rules/SuperfluousDisableCommandRule.swift

```diff
@@ -11,7 +11,17 @@ public struct SuperfluousDisableCommandRule: ConfigurationProviderRule, SourceKi
             SwiftLint 'disable' commands are superfluous when the disabled rule would not have triggered a violation \
             in the disabled region. Use " - " if you wish to document a command.
             """,
-        kind: .lint
+        kind: .lint,
+        nonTriggeringExamples: [
+            Example("↓let abc:Void // swiftlint:disable:this colon"),
+            Example("↓// swiftlint:disable colon (If it is enabled in the configuration file)"),
+            Example("↓// swiftlint:enable colon (If it is disabled in the configuration file)")
+        ],
+        triggeringExamples: [
+            Example("↓let abc: Void // swiftlint:disable:this colon"),
+            Example("↓// swiftlint:disable colon (If it is disabled in the configuration file)"),
+            Example("↓// swiftlint:enable colon (If it is enabled in the configuration file)")
+        ]
```

Comment:
I'd like to make this a bit clearer, in that "non-triggering" really shows an example in which the `colon` rule would trigger while the command disables it. Similarly, the "triggering" case presents the same example but now with the rule triggering which draws the `disable` command useless. That's what `superfluous_disable_command` is there for and thus the examples should depict.

I also think it's clear that the `colon` rule must be enabled in the first place. We might not need to mention this in extra comments.

Contrary to one of my last review comments, violations markers are not helpful here as they would appear at a confusing location. The exact location anyway doesn't matter so much for this rule in my opinion. Sorry for the back and forth.

That said, my suggestion would be like:

```suggestion
        nonTriggeringExamples: [
            Example("let abc:Void // swiftlint:disable:this colon"),
            Example("""
                // swiftlint:disable colon
                let abc:Void
                // swiftlint:enable colon
                """)
        ],
        triggeringExamples: [
            Example("let abc: Void // swiftlint:disable:this colon"),
            Example("""
                // swiftlint:disable colon
                let abc: Void
                // swiftlint:enable colon
                """)
        ]
```